catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -184,6 +192,10 @@ public void addFields(Metacard metacard, SolrInputDocument solrInputDocument)
             if (metacard.getAttribute(ad.getName()) != null) {
                 List<Serializable> attributeValues = metacard.getAttribute(ad.getName())
                         .getValues();
+                if (ad.getType()
```

Comment:
Why is this here, and why is not already handled by `addToFieldsCache`? It seems redundant.